library/psa_crypto.c

```diff
@@ -6486,14 +6488,29 @@ psa_status_t psa_key_derivation_output_key_ext(
     return status;
 }
 
+psa_status_t psa_key_derivation_output_key_ext(
```

Comment:
We only worry about C++ in public headers, because they can be read by a C++ compiler. We do not intend the library to be built by a C++ compiler (and there are already many reasons why it wouldn't work).